scholarx/archive/index.html

```diff
@@ -138,7 +138,7 @@ <h6>UNIVERSITY</h6>
                 </ul>
               </div>
         </div>
```

Comment:
Add a new space after this div too @janithlahirukariyawasam 
```suggestion
          </div>
```

Reply:
@janithlahirukariyawasam can you add one more space there